.gitignore

```diff
@@ -12,3 +12,4 @@
 *.test
 tools/functional-tester/docker/bin
 hack/tls-setup/certs
+.idea
```

Comment:
Intellij Idea IDE specific files. Not sure whether it's right way to do it or not?